### PR TITLE
fix laravel command not found

### PR DESCRIPTION
**Pull request by mmhyamin**
Issue: http://stackoverflow.com/a/27915343/3060426

### CODE REVIEW

**Comment by pvalena** [comment]
My .bash_profile already contains `export PATH` and .bashrc seems like a strange place for path.
Could you elaborate on why is this an issue?

File: start/sw/web-app/laravel5.md [hunk 1]
```diff
@@ -58,10 +58,10 @@ $ composer -V
 $ composer global require "laravel/installer"
 # Place the ~/.config/composer/vendor/bin directory in your PATH so the laravel 
 # executable can be located by your system.
-$ echo 'export PATH="$PATH:$HOME/.config/composer/vendor/bin"' >> ~/.bash_profile
```
Comment (pvalena) [inline start/sw/web-app/laravel5.md]: You do not need the `export` here. PATH should be already exported on all UNIX systems.

For immediate change in current terminal you could run the command `source !$` instead, which sources ~/.bash_profile.

Comment (mmhyamin) [inline start/sw/web-app/laravel5.md]: @pvalena could you please take a look at the [stackoverflow](http://stackoverflow.com/questions/26376516/laravel-php-command-not-found) issue?

Comment (pvalena) [inline start/sw/web-app/laravel5.md]: @mmhyamin I sure already did, but I think the answer you have sent is not the correct one (FTR it's not the accepted one) in this case.
Could you check my proposed solution?
Please clarify the issue you're having if it is not resolved by it (`source !$`, right after the `PATH=". . .`).

Comment (pvalena) [inline start/sw/web-app/laravel5.md]: We'll resolve the issue later.

**Pull request merged**